src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitsStorage = new HashMap<>();
```

Comment:
The access to 'fruitsStorage' should be encapsulated. It's a bad practice to make your storage map public and static. Instead, provide public static methods to interact with the storage, such as 'addFruit', 'getFruitQuantity', 'removeFruit', etc., and make the 'fruitsStorage' map private.